smf/sled-agent/non-gimlet/config-rss.toml

```diff
@@ -41,7 +41,7 @@ external_dns_zone_name = "oxide.test"
 # the DNS domain delegated to the rack by the customer. Each of these addresses
 # must be contained in one of the "internal services" IP Pool ranges listed
 # below.
-external_dns_ips = [ "192.168.1.20", "192.168.1.21" ]
+external_dns_ips = [ "10.85.0.20", "10.85.0.21" ]
```

Comment:
Nit: Is this necessary? Or just a local change that made it into this PR?

Reply:
Local change that slipped into the PR, my bad 😂